index.html

```diff
@@ -1688,6 +1688,16 @@ <h3>Names and Descriptions</h3>
 <a href="#language-and-base-direction"></a> for further information.
         </p>
 
+        <p class="note">
+The `@direction` attributes in the examples below are not technically required
```

Comment:
"technically" doesn't seem needed here.
```suggestion
The `@direction` attributes in the examples below are not required
```